src/tests/work-item/work-item-list/unauthorizeduser.spec.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * POC test for automated UI tests for ALMighty
+ *  Story: Display and Update Work Item Details
+ *  https://github.com/almighty/almighty-core/issues/296
+ *
+ * Note on screen resolutions - See: http://www.itunesextractor.com/iphone-ipad-resolution.html
+ * Tests will be run on these resolutions:
+ * - iPhone6s - 375x667
+ * - iPad air - 768x1024
+ * - Desktop -  1920x1080
+ *
+ * beforeEach will set the mode to phone. Any tests requiring a different resolution will must set explicitly.
+ *
+ * @author nverma
+ */
+
+var WorkItemListPage = require('./work-item-list.page'),
+    WorkItemDetailPage = require('./work-item-detail.page'),
+  testSupport = require('./testSupport');
+
+describe('Work item list', function () {
+  var page, items, startCount, browserMode,detailPage;
+
+  beforeEach(function () {
+    testSupport.setBrowserMode('phone');
+    page = new WorkItemListPage(false);
+    detailPage=new WorkItemDetailPage(1);
+  });
+
+/*Test Quick add work item should not be visible*/ 
+  it('Test Quick workitem visible without authorization - phone.', function () { 
+    page.clickLogoutButton().click();
+    expect(page.quickAddbuttonById().isPresent()).toBeFalsy();
+   });
+/*Test user should not be able to Delete any work item*/ 
+  it('Test user should not be able to Delete any work item - phone.', function () {
+    expect(page.KebabButtonById().isPresent()).toBe(false);
+   });
+/*Test user should not be able update the work item title */
+  it('Test user should not be able update the work item title - phone.', function () {
+    testSupport.setBrowserMode('phone');
+    page.workItemByIndex(1).click();
+    expect(detailPage.workItemTitleEditIconById().isPresent()).toBeFalsy();
+   
+   });
+/*Test user should not be able update the work item description */
+  it('Test user should not be able update the work item description - phone.', function () {
+    testSupport.setBrowserMode('phone');
```

Comment:
@nimishamukherjee , can add this check in the same case! for title and description both.